src/game.rs

```diff
@@ -90,14 +90,13 @@ impl Region {
     }
 }
 
-pub fn generate_board(number_of_players: usize) -> Board {
```

Comment:
Can this be changed, to use the resource, instead of the underlying prng engine?

Reply:
I thought so, but I guess better solution would be something similar to `PrngPlugin` with initialization in `build` function.

Reply:
Ideally, any system(resource, or entity) that needs to pull from RNG, should either pull from an existing Prng resource, or have a new one added to the PrngPlugin so that there is a dedicated resource for that consumer. For example reinforcement of dice at the end of a turn could either pull from the map Prng, or have its own resource created. One major important detail is that once a Prng resource is added, the order is never changed, and if that resource does get deprecated, the prng seed that it used to bootstrap would be continue to be read and then trashed(so later resources don't end up shifting their offset in the Prng stream).

Reply:
Ignore the above. I had misread the code. The resource being inserted is the variable from above, not a default version of the resource.